fem/bilininteg_convection_ea.cpp

```diff
@@ -230,7 +230,7 @@ void ConvectionIntegrator::AssembleEA(const FiniteElementSpace &fes,
                                       const bool add)
 {
    AssemblePA(fes);
-   const int ne = fes.GetMesh()->GetNE();
+   ne = fes.GetMesh()->GetNE();
```

Comment:
I think the intent was to bring it locally in order to avoid pulling the class to use its `ne` on the device.

Reply:
```suggestion
```

Reply:
I don't see another place where `ne` is set. Have I missed something?

Reply:
Good point! I didn't realize that the constructor was leaving leaving these fields uninitialized. My bad.

Reply:
Actually, I talked too quickly, this field is initialized by `AssemblePA(fes);` the line above.

Reply:
> I think the intent was to bring it locally in order to avoid pulling the class to use its `ne` on the device.

This type of thinking is new to me so I apologize if the answer is obvious. It looks like the vector `pa_data` will be needed from the class. Would also using `ne` from the class be more demanding in some way?